convert2rhel/unit_tests/subscription_test.py

```diff
@@ -494,6 +494,10 @@ def test_register_system_fail_interactive(self, tool_opts, monkeypatch, caplog,
     @pytest.mark.rhsm_returns((KeyboardInterrupt("bang")))
     def test_register_system_keyboard_interrupt(self, tool_opts, monkeypatch, caplog, mocked_rhsm_register):
         """Test that we stop retrying if the user hits Control-C.."""
+
+        print(os.environ)
```

Comment:
Unfortunately there's nothing more than in a docker image when executed locally:
```
 {'LANG': 'C', 'APP_MAIN_DEPS': 'python-six     dbus-python     pexpect', 'LC_ALL': 'C', 'PYTHON': 'python2', 'HOSTNAME': 'c80479088321', 'PYTHONDONTWRITEBYTECODE': '1', 'PYTEST_CURRENT_TEST': 'convert2rhel/unit_tests/subscription_test.py::TestRegisterSystem::test_register_system_keyboard_interrupt (call)', 'APP_DEV_DEPS': 'requirements/centos7.requirements.txt', 'URL_GET_PIP': 'https://bootstrap.pypa.io/pip/2.7/get-pip.py', 'HOME': '/home/app', 'PIP': 'pip', 'PATH': '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'}
 ```

Reply:
mbocekm later spotted that this test was calling `register_system()` twice and one of those was outside of the `with pytest.raises()`.  I've removed the extra `register_system()` and the unit_tests now pass.